### ISSUE TITLE
🐛 BUG: incorrect HSL syntax results in css [scss] file not found

### ISSUE THREAD

**Issue by IanVS** [snowpackjs/astro#870]
### What package manager are you using?

npm

### What operating system are you using?

Mac

### Describe the Bug

I'm new to Astro and I was trying to convert from css to scss, but kept getting `Not found /global.css`. Based on the portfolio example I thought I was doing everything correctly.  Eventually I traced the problem down to some syntax like `$gray: hsl(214, 15, 50);`.  There was no output warning me about the failing syntax, only:

```
[16:42:04] [snowpack] Build Result Error: There was a problem with a file build result.
[static] Not found /global.css
```

### Steps to Reproduce

1. `npm init astro` using template portfolio
2. In `public/global.scss` add a line at the top: `$black: hsl(228, 78, 7);`
3. `npm start`
4. Open the website
5. Error! Describe what went wrong (and what was expected instead)...

```
[snowpack] Build Result Error: There was a problem with a file build result.
[static] Not found /global.css
```

Ideally there would be some output from sass explaining what failed.

**Comment by jasikpark** [comment]
To act as the fancy error message you are requesting, I think you need `%` on the second and third inputs to `hsl()` at least in CSS (though I'm sure CSS will deal with it being wrong at the same time). 

+1 on this request though! It'd be good to surface the actual compiler error message that the SASS compiler is outputting. I wonder if you try running your failing build with `--verbose` does it show you the error message?

**Comment by drwpow** [comment]
This is an issue with Snowpack’s Sass plugin where the error message is swallowed, similar to some other instances. This will be fixed with the new compiler.

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
